### PR TITLE
Replace UnifiedSet.newSet(initialCapacity) with Sets.mutable.withInitialCapacity().

### PR DESCRIPTION
**Pull request by motlin**

### CODE REVIEW

File: eclipse-collections/src/main/java/org/eclipse/collections/impl/set/mutable/MultiReaderUnifiedSet.java [hunk 1]
```diff
@@ -109,7 +109,7 @@ public static <T> MultiReaderUnifiedSet<T> newSet()
 
     public static <T> MultiReaderUnifiedSet<T> newSet(int capacity)
     {
-        return new MultiReaderUnifiedSet<>(UnifiedSet.newSet(capacity));
+        return new MultiReaderUnifiedSet<>(Sets.mutable.withInitialCapacity(capacity));
```
Comment (donraab) [inline eclipse-collections/src/main/java/org/eclipse/collections/impl/set/mutable/MultiReaderUnifiedSet.java:112]: This should remain `UnifiedSet`.

**Review by donraab — CHANGES_REQUESTED** [review]
One change requested.

File: eclipse-collections/src/main/java/org/eclipse/collections/impl/set/mutable/MultiReaderUnifiedSet.java (hunk 1 above)
Comment (motlin) [reply eclipse-collections/src/main/java/org/eclipse/collections/impl/set/mutable/MultiReaderUnifiedSet.java]: Oops, fixed.

**Review by donraab — APPROVED** [review]
LGTM

**Pull request merged**